helm-files.el

```diff
@@ -5269,6 +5270,13 @@ Find inside `require' and `declare-function' sexp."
                ;; If require use third arg, ignore it,
                ;; always use library path found in `load-path'.
                (cl-second (split-string (match-string 0 sexp))))))
+            ((and sexp (string-match "use-package .+[^)]" sexp))
+             (find-library-name
+              (replace-regexp-in-string
+               "'\\|)\\|(" ""
```

Comment:
And then use this group e.g.`(match-string 1)` instead of using `replace-regexp-in-string`.

Reply:
Some thing like this:
```elisp
(cond ((and sexp (string-match "require +'\\([^) ]*\\)" sexp))
             (find-library-name (match-string 1 sexp)))
            ((and sexp (string-match "use-package +\\([^) ]*\\)" sexp))
             (find-library-name (match-string 1 sexp)))
            ((and sexp
                  (string-match
                   "declare-function +\\([^) ]*\\) \"\\(ext:\\)*\\([^\"]*\\)"
                   sexp))
             (find-library-name (match-string 3 sexp)))
            (t nil))
```

Reply:
I Just updated the code.

Reply:
I can refactor the other clauses as well.